sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,56 +1,116 @@
 # Release History
-
-## not release yet (2021-3-24)
+## 1.0.0-beta.9 (Unreleased)
```

Comment:
We have only three days left to release beta 9 from this branch. Might as well make it 03-29
```suggestion
## 1.0.0-beta.9 (2021-03-29)
```

Reply:
Sounds good, I'll change the date back.